examples/ssh-host-agave-validator-python/__main__.py

```diff
@@ -0,0 +1,57 @@
+"""An SSH Python Pulumi program"""
+
+import pulumi
+import pulumi_tls as tls
+import pulumi_svmkit as svmkit
+
+ssh_key = """-----BEGIN OPENSSH PRIVATE KEY-----
+-----END OPENSSH PRIVATE KEY-----"""
```

Comment:
Can you look at moving this to the Pulumi config and add a simple readme that describes encrypting the ssh key with pulumi?